feature-libs/storefinder/core/service/geolocation.service.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { PointOfService } from '@spartacus/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class GeolocationService {
```

Comment:
Please vote for a name of that service (or give your own proposal):
👎🏻 GeolocationService
👍🏻 StoreLocationService
❤️ StoreGeolocationService
👀 PointOfServiceLocationService
😄 StoreMapLinkService
🚀 StoreCoordinatesService
🎉 LocationUrlService